### ISSUE TITLE
💡 Feature Request - Enable service health alerts on provisioned subscriptions

### ISSUE THREAD

**Issue by sebassem** [Azure/bicep-lz-vending#60]
## Describe the solution you'd like

Enable service health alerts on provisioned subscriptions to enable better workload reliability through various alerts generated from service health

## Describe alternatives you've considered

N/A

## Additional context

Add any other context or screenshots about the feature request here. 📷

**Comment by tulpy** [comment]
Im assuming that the ALZ AMBA guidance [here](https://azure.github.io/azure-monitor-baseline-alerts/patterns/alz/) would address the Service Health requirement?

**Comment by sebassem** [comment]
> Im assuming that the ALZ AMBA guidance [here](https://azure.github.io/azure-monitor-baseline-alerts/patterns/alz/) would address the Service Health requirement?

Yes, good point for customers we are using AMBA and applying the policies at the management group level
